docs/home/installing-tracking-script.md

```diff
@@ -0,0 +1,17 @@
+## How to install the tracking script
+
+After installation and setup of the cron jobs you're ready to begin tracking contacts. You need to add a simple piece of JavaScript to the websites for each site you wish
```

Comment:
```suggestion
After installation and setup of the cron jobs you're ready to begin tracking Contacts. You need to add a piece of JavaScript to the websites for each site you wish
```

I think we should add a link to the cron jobs page here, also Contacts should be capitalised and we try not to use terms like 'simply' (Vale flags as a problem to fix)

Reply:
okay